lib/internal/test_runner/test.js

```diff
@@ -108,6 +109,11 @@ class TestContext {
     return this.#test.mock;
   }
 
+  get fakeTimers() {
```

Comment:
I would expect `fakeTimers` to be part of the existing `mock` API. That would make the code to integrate it with the test runner simpler (no changes in this file or `lib/test.js`), while also having less of a performance impact (as it's currently written, the impact is small, but it could be smaller). It would also let the API have a slightly nicer (IMO) name of `t.mock.timers` instead of `t.fakeTimers`. The fake timers could also be auto-restored after each test like the other mock functionality.

Reply:
The way it works in sinon and how Jest uses it is that:

 - You can either get a copy of fresh fake timers (a "clock") which doesn't modify globals
 - You can `.install` which overrides the globals like setTimeout/setInterval etc.

This is useful since if Node provided an API for the fake timers (not in this PR!) that worked for cases timers are injected.

Reply:
+1 on t.mock.timers over t.fakeTimers btw

Reply:
moved to the mock API